assets/sass/components/_alerts.scss

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I wonder if this file should be named to match the component, so `_error_summary.scss`, or something similar?